docs/commands/data.json

```diff
@@ -191,10 +191,6 @@
         "description": "Name of the resource group to create new storage account when it does not exist",
         "required": true
       },
-      {
```

Comment:
Was this intentionally made to be modified?

Reply:
@NathanielRose we have a docs regeneration pre-commit hook. That is updating the docs

Reply:
@NathanielRose 
this is generated by pre-commits hooks. it is ok